modules/ROOT/pages/bare-metal.adoc

```diff
@@ -174,6 +174,32 @@ A useful approach is to script generating a per-machine `.iso`. If you have a ha
 
 Alternatively, instead of generating per-machine ISOs, you can have a `--pre-install` script run a privileged container which inspects the target system and writes an appropriate https://coreos.github.io/coreos-installer/customizing-install/#config-file-format[installer config] to `/etc/coreos/installer.d`.
 
+=== Installing on iSCSI
+
+To install CoreOS on an iSCSI boot device, follow the same
+steps as described above to get the live environnement, simply adding the correct kernel arguments.
+
+
+- Mount the iscsi target from the live environnement :
+[source, bash]
+----
+sudo iscsiadm -m discovery -t st -p 127.0.0.1
+sudo iscsiadm -m node -T iqn.2023-10.coreos.target.vm:coreos -l
+----
+
+- run `coreos-installer`:
+[source, bash]
+----
+sudo coreos-installer install \
```

Comment:
Maybe let's just include both the manual and iBFT examples in full? There's a syntax for labeling each one:

```adoc
.Installing to an iSCSI target with iBFT
[source, bash]
---
...
---

.Installing to an iSCSI target with manual configuration
[source, bash]
---
...
---
```

And then let's also note that all this can also be set at `coreos-installer iso customize` or `coreos-installer pxe customize` time? (We can use `--pre-install` to run the `iscsiadm` command, and then `--dest-device` and `--dest-karg-append` for the rest -- though would be good to sanity-check that this works as expected.)